app/cdap/components/DataPrepHome/index.js

```diff
@@ -300,6 +300,7 @@ export default class DataPrepHome extends Component {
           <Connections
             mode={mode || 'ROUTED_WORKSPACE'}
             onWorkspaceCreate={mode === 'INMEMORY' ? null : this.onWorkspaceCreate}
+            allowDefaultConnection={true}
```

Comment:
@njbriggs is this correct? Should DataPrepHome use the default connection?

Reply:
Yes. `DataPrep` is Wrangler